dataactbroker/handlers/fileHandler.py

```diff
@@ -1225,6 +1171,44 @@ def get_d_file_url(task_key, file_type_name, cgac_code, start_date, end_date):
         url = CONFIG_BROKER["".join([file_type_name, "_url"])].format(cgac_code, start_date, end_date, callback)
         return url
 
+    @staticmethod
+    def create_generation_task(job_id):
```

Comment:
Nice extraction. Note that this could be a straight function (instead of a staticmethod) for further decoupling